client/src/components/Donation/DonateForm.tsx

```diff
@@ -328,7 +330,13 @@ class DonateForm extends Component<DonateFormProps, DonateFromComponentState> {
 
   render() {
     const {
-      donationFormState: { processing, success, error, redirecting },
+      donationFormState: {
+        processing,
+        success,
+        error,
+        redirecting,
+        isSignedIn
+      },
       isMinimalForm
```

Comment:
isSignedin is already defined as a prop so we don't need to include it in the donationFromState.
You could destruct it similar to isMinimalForm here and pass it to renderCompletion.